src/components/autocomplete/js/autocompleteController.js

```diff
@@ -236,7 +243,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
    */
   function onListLeave () {
     noBlur = false;
-    if (!hasFocus) ctrl.hidden = true;
+    ctrl.hidden = shouldHide();
```

Comment:
I just want to confirm that this works properly.  I believe at the time this logic was added, we had to explicitly set it to true.